x-pack/test/security_solution_cypress/es_archives/process_ancestry/data.json

```diff
@@ -0,0 +1,277 @@
+{
+  "type": "doc",
```

Comment:
As this document is from an alert, why are we creating a rule on the main test to generate an alert?

Reply:
You are right.. I was pushing it to wrong index, I have correct that and removed the rule creation step.